pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/FunctionCallAstNode.java

```diff
@@ -82,8 +82,8 @@ public AggregationInfo buildAggregationInfo() {
       expression = "*";
     } else {
       List<? extends AstNode> children = getChildren();
-      if (children == null || children.size() != 1) {
-        throw new Pql2CompilationException("Aggregation function expects exact 1 argument");
+      if (children == null || (children.size() != 1 && !_name.equalsIgnoreCase("distinct"))) {
```

Comment:
Same here?

Reply:
sorry, it was bad merged. removed changes